app/views/coronavirus_form/need_help_with.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for :title do %><%= t("coronavirus_form.questions.need_help_with.title") %><% end %>
+<% content_for :meta_tags do %>
+  <meta name="description" content="<%= t("coronavirus_form.questions.need_help_with.title") %>" />
+<% end %>
+
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", { href: previous_path } %>
+<% end %>
+
+<%= form_tag({},
+  "data-module": "track-coronavirus-form-find-support-need-help-with",
+  "data-question-key": "need_help_with",
+  "id": "need_help_with",
+  "novalidate": "true"
+) do %>
+
+<%= render "govuk_publishing_components/components/checkboxes", {
+  heading: t("coronavirus_form.questions.need_help_with.title"),
+  is_page_heading: true,
+  name: "need_help_with[]",
+  error_message: error_items("need_help_with"),
```

Comment:
Unfortunately, checkboxes API is inconsistent with the radios API, so we'll need to use `error` to show the error message at the component-level.
```suggestion
  error: error_items("need_help_with"),
```